src/wx/dialogs/speedup-config.h

```diff
@@ -0,0 +1,32 @@
+#ifndef VBAM_WX_DIALOGS_SPEEDUP_CONFIG_H_
+#define VBAM_WX_DIALOGS_SPEEDUP_CONFIG_H_
+
+#include "wx/config/bindings.h"
+#include "wx/config/option-proxy.h"
+#include "wx/dialogs/base-dialog.h"
+
+namespace dialogs {
+
+// Manages the Speedup/Turbo configuration dialog.
+class SpeedupConfig : public BaseDialog {
+public:
+    static SpeedupConfig* NewInstance(wxWindow* parent);
+    ~SpeedupConfig() override = default;
+
+    void ToggleSpeedupFrameSkip();
+    void ToggleSpeedupMute();
+    void SetSpeedupThrottle(wxCommandEvent& evt);
+
+private:
+    // The constructor is private so initialization has to be done via the
+    // static method. This is because this class is destroyed when its
+    // owner, `parent` is destroyed. This prevents accidental deletion.
+    SpeedupConfig(wxWindow* parent);
+
+    bool prev_frame_skip_cb_ = OPTION(kPrefSpeedupThrottleFrameSkip);
```

Comment:
Please, move the instantiation to the constructor implementation. It's fine to use them in the definition for simple constructors but this is a complex instantiation.